cpp/tests/utilities/column_utilities.cu

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "column_utilities.cuh"
+
+#include <cudf/column/column_view.hpp>
+#include <cudf/table/row_operators.cuh>
+#include <cudf/table/table_device_view.cuh>
+#include <tests/utilities/cudf_gtest.hpp>
+
+#include <thrust/equal.h>
+
+#include <gmock/gmock.h>
+
+namespace cudf {
+namespace test {
+
+// Verify elementwise equality
+void expect_columns_equal(cudf::column_view lhs, cudf::column_view rhs) {
```

Comment:
While this method of writing `EXPECT_` statements is better than the old `column_wrapper<>::operator==()` one, can you add an option to print the columns when they don't match. It's been very useful to have printouts show up in CI and while debugging using tests.

Reply:
Yeah, I intend to do that in a future improvement.